doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_suit_recovery.rst

```diff
@@ -0,0 +1,183 @@
+.. _ug_nrf54h20_suit_recovery:
+
+SUIT manufacturer application recovery
+######################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+During the device lifetime, the device may encounter various issues that can lead to boot failure.
+In such cases the device needs a way to recover from the failure and continue to operate.
+
+In SUIT, this is solved by running a specially prepared recovery firmware.
+Although the recovery firmware needs some additional space in the device, it is a highly recommended feature for all devices that use SUIT.
+
+This document describes scenarios in which a nRF54H20 device using SUIT can enter recovery mode and how the recovering process works.
+
+When is recovery mode entered?
```

Comment:
```suggestion
When is recovery mode entered
```
I'm pretty sure you're not asking a question here, but explaining when this happens, so ? is not needed.